tests/testing_utils/fixtures.py

```diff
@@ -247,7 +247,7 @@ def func():
     return func
 
 
-@pytest.fixture
+@pytest.fixture()
```

Comment:
why?

Reply:
removed to keep things consistent, it works both ways.